contrib/inventory/openstack.py

```diff
@@ -61,7 +61,8 @@
 import shade
 import shade.inventory
 
-CONFIG_FILES = ['/etc/ansible/openstack.yaml', '/etc/ansible/openstack.yml']
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__)) + '/clouds.yaml'
```

Comment:
This should also look for `openstack.yml` and also account for `yaml` and `yml` file extensions.